tests/test_cache.py

```diff
@@ -0,0 +1,12 @@
+from unittest import TestCase
+from canonicalwebteam.http import create_cached_session
+from datetime import timedelta
+
+
+class TestCachedSession(TestCase):
+    def test_custom_heuristic(self):
+        session = create_cached_session(fallback_cache_duration=10)
+        adapter = session.get_adapter("http://")
+        heuristic = adapter.heuristic
+        delta = heuristic.delta
+        self.assertEqual(delta, timedelta(seconds=10))
```

Comment:
@b-m-f and @tbille - did you see my comment about putting the `main` function lines back? I think including them is better practice. Not everyone coming to our project might know of `unittest`, they might not have the `unittest` binary installed, and they might not know how to do `python -m unittest`. I think it is inherently more thorough to allow our tests to be run directly with `python3 test.py`.